packages/hooks/src/usePaymentAgentTransferVisible.ts

```diff
@@ -0,0 +1,18 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const usePaymentAgentTransferVisible = () => {
+    const { client } = useStore();
+    const { is_authorize } = client;
+
+    const { data, ...rest } = useFetch('get_settings', { options: { enabled: is_authorize } });
+
+    const is_payment_agent_transfer_visible = Boolean(data?.is_authenticated_payment_agent);
```

Comment:
Maybe we could use `!!data?.is_authenticated_payment_agent`

Reply:
Or we could stick with `Boolean(...)` for readability

Reply:
@rostislav-deriv that was my concern when i decided to use Boolean(). Also i found an article in git hub https://gist.github.com/arthurvi/66cb1e2bcfc92f99f465e0db04264367 !! vs Boolean(). There also mentiones about readability and other things...